devel/gk/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gk
+version                 1.0.6
+
+categories              devel
+platforms               darwin
+license                 CC-BY-3.0-US
+installs_libs           no
+maintainers             sergiolms \
+                        alejandroSuch \
+                        JBUinfo
```

Comment:
Each Github username should have its @ sign, and be surrounded by curly braces.  Additionally a port should also be `openmaintainer` unless there is a reason why only you and the listed maintainers are the only ones allowed to update the port:

```
maintainers             {@sergiolms} \
                        {@alejandroSuch} \
                        {@JBUinfo} \
                        openmaintainer

Reply:
curly brackets are needed to group a GitHub handle and (obfuscated) e-mail address - the latter is missing here.